modules/dxp/apps/analytics/analytics-reports-web/src/main/resources/META-INF/resources/css/main.scss

```diff
@@ -53,3 +53,7 @@
 		margin-bottom: 0.5rem;
 	}
 }
+
+.dropdown-menu {
+	max-height: 295px;
+}
```

Comment:
@dgarciasarai: Shouldn't we be scoping this so that it doesn't affect all drop-downs?

I'll leave the commenting on your other suggestion to @marcoscv-work and @edalgrin.

Reply:
As far as I know (and as far as I saw trying to solve this), the dropdown items area is created outside time-space sidebar limits. My first idea was adding this inside `.lfr-analytics-reports-panel` but it didn't work. I'm open to other ideas to make this more beautiful 😄

Reply:
You can pass custom classes to the `dropdown-menu` in `ClayDropDown` with:
```
menuElementAttrs={{
  className: 'some-class',
}}
```
The strange part here is Clay CSS `dropdown-menu` has `min-height: 40px` and `max-height: 500px`. If the number of items are taller than `500px`, it should overflow horizontally. There must be some CSS in translation that are overwriting these. In any case, `menuElementAttrs` should solve this issue.

Reply:
> The strange part here is Clay CSS dropdown-menu has min-height: 40px and max-height: 500px. If the number of items are taller than 500px, it should overflow horizontally vertically. There must be some CSS in translation that are overwriting these. In any case, menuElementAttrs should solve this issue.

That is happening in the translation component, but we are in a sidebar, so we want to make it overflow vertically much before 500px. I'm gonna try with `menuElementAttrs`. Thanks!

Reply:
See 5547236